docs/developers/dependencies.md

```diff
@@ -0,0 +1,55 @@
+
+---
+title: Dependency Management
+---
+
+Dependencies are common. You need to hook into another plugin. You don't want to write the same code
+someone else has already solved. Whatever you do, you need a way to manage your dependencies effectively.
```

Comment:
How about we change this away from being what dependencies are (as this is really developer docs) and more toward how to use them with velocity. @astei any thought on this?

Reply:
I felt that it was important to urge relocation for external dependencies. On Bukkit there are way too many plugins which don't, resulting in lots of classloading confusion and conflicts. Plugin developers often don't understand, since the issue only occurs when another plugin with the same failure to relocate is present.